justfile

```
@@ -8,7 +8,7 @@ pre-push: fmt github-ci
 fast-pre-push: fmt update-flakefiles test check-examples typescript-check test-ts
 
 # Run checks that we can’t yet run via the flake
-github-ci: codegen test check-isolated-garn check-examples test-ts
+github-ci: (update-flakefiles "true") codegen test check-isolated-garn check-examples test-ts
```

What are your thoughts on having a step that verifies that after running all the tests the repo is still in a clean state? This could simplify `update-flakefiles`. Although I don't feel strongly, TIOLI!

```suggestion
github-ci: update-flakefiles codegen test check-isolated-garn check-examples test-ts check-not-dirty

check-not-dirty:
  git diff-files --quiet
```